### ISSUE TITLE
Install Error

### ISSUE THREAD

**Issue by HippyCraig** [VanishsVault/apsjournal#1]
I tried using:

https://github.com/VanishsVault/apsjournal/releases/latest/download/module.json

I got the following:

Error: Unable to load valid module manifest data from "https://github.com/VanishsVault/apsjournal/releases/latest/download/module.json"
The requested manifest at "https://github.com/VanishsVault/apsjournal/releases/latest/download/module.json" did not provide module manifest data.
    at Module.installPackage (file:///home/hcfoundryuser/foundryv10/resources/app/dist/packages/views.mjs:1:1620)
    at processTicksAndRejections (node:internal/process/task_queues:96:5)
    at async SetupView.handlePost (file:///home/hcfoundryuser/foundryv10/resources/app/dist/server/views/setup.mjs:1:1980)

**Comment by AmazingVanish** [comment]
Yeah, in order to use the module, you'll have to download the package and install it manually. I don't have this setup for proper distribution yet, Sorry.

**Comment by HippyCraig** [comment]
ok no worries I will do that

**Comment by HippyCraig** [comment]
I have it installed an so far so good.  I tried a few of the layouts and it looks good, and the HTML looks right.

**Comment by AmazingVanish** [comment]
@HippyCraig Here is the correct repo. ;)
